server/controllers/articles.js

```diff
@@ -1 +1,129 @@
 import Article from "../models/Articles";
+export const getAllArticles = async (req, res, next) => {
+    try {
+        const articles = await Article.find();
+
+        return res.status(200).json({
+            success: true,
+            status: 200,
+            count: articles.length,
+            data: articles
+        });
+    } catch (err) {
+        return res.status(500).json({
+            success: false,
+            status: 500,
+            error: 'Server error'
+        });
+    }
+}
+
+export const createArticle = async (req, res, next) => {
+        const newArticle = new Article (
+            {
+                title: req.body.title,
+                description: req.body.description,
+                picURL: req.body.picURL,
+                content: req.body.content,
+                date: ""
+            }
+        )
+        newArticle.save(function(err, prod){
+            if(!err){
+                return res.status(200).json({
+                    success: true,
+                    status: 200,
+                    message: 'Article created successfully',
+                    data: prod
+                });
+            } else{
+                return res.status(500).json({
+                    success: false,
+                    status: 500,
+                    error: 'Server error',
+                    data: err
+                });
+            }
+        });
+        
+}
+
+export const getOneArticle = async (req, res, next) => {
+    try {
+        const article = await Article.findById(req.params._id);
+console.log('>>>>>',req.params._id);
```

Comment:
Could you please remove this unnecessary console.log?

Reply:
Yeah sure!